assert/mock.go

```diff
@@ -16,6 +16,11 @@ func (m *mockT) Errorf(format string, args ...interface{}) {
 	m.message = fmt.Sprintf(format, args...)
 }
 
+func (m *mockT) Error(args ...interface{}) {
+	m.failed = true
+	m.message = fmt.Sprint(args...)
+}
```

Comment:
I'm surprised here, it should be


```suggestion
func (m *mockT) Error(format string, args ...interface{}) {
	m.failed = true
	m.message = fmt.Sprintf(format, args...)
}
```

You can also use the `if len(args) > 0`

Reply:
Wouldn't this be the signature of Errorf, not Error?